code/game/machinery/cryopod.dm

```diff
@@ -328,6 +335,12 @@
 				to_chat(O.owner.current, "<span class='warning'>You get the feeling your target is no longer within your reach...</span>")
 			qdel(O)
 
+	//Same for contract-based objectives.
+	for(var/c in GLOB.all_antag_contracts)
```

Comment:
When for( in ) loops, each element from GLOB.all_antag_contracts is being put in c, so in same line it can do var/datum/antag_contract/contract/c

Reply:
If you put a type into `in` loop, it will check every entry in the list for that type. If you know what type are the entries in the list already, doing it this way is a tiny bit faster.

Tgstation used to enforce this shit, so I'm doing it out of habit.

Reply:
Understood, thanks !